lambda/src/main/resources/db/migration/V170__add_restrictions_on_use_to_file_properties.sql

```diff
@@ -0,0 +1,7 @@
+-- Add restrictions_on_use property
+
+INSERT INTO "FileProperty" ("Name", "Description", "FullName", "PropertyType", "Datatype", "Editable", "MultiValue",
+                            "PropertyGroup", "AllowExport", "ExportOrdinal")
+VALUES ('RestrictionsOnUse', NULL, 'restrictions_on_use', 'Supplied', 'text', true, false, 'OptionalMetadata', true, 155);
```

Comment:
If `AllowExport` is set to `true` then this column will appear in the `file-metadata.csv` (https://github.com/nationalarchives/tdr-consignment-export/blob/master/bagit-export/src/main/scala/uk/gov/nationalarchives/consignmentexport/BagAdditionalFiles.scala#L26) This should not be done until Data Services are ready for the new column.

Should set this to `false` for now. When Data Services are ready it should be set to `true` in a separate migration.

Reply:
changed to false